packages/entryPoints/index.ts

```diff
@@ -303,6 +303,19 @@ export async function startPreview() {
     wsScene = `${location.protocol === 'https:' ? 'wss' : 'ws'}://${document.location.host}/?scene`
   }
 
+  getPreviewSceneId().then((sceneData) => {
```

Comment:
can we use async/await here instead of promises.then ?
```ts
const sceneData = await getPreviewSceneId();
if (sceneData.sceneId) {
  const { unityInterface } = await ensureUnityInterface()
  unityInterface.SetKernel.....
}
```

Reply:
👍 I will check if doing that change has an impact or not in the loading time of the preview scene. If there is any then I will change it as you suggest :)